plugin/azure/README.md

```diff
@@ -0,0 +1,83 @@
+# azure
+
+## Name
+
+*azure* - enables serving zone data from Microsoft Azure DNS service.
+
+## Description
+
+The azure plugin is useful for serving zones from resource record
+sets in Microsoft Azure DNS. The `azure` plugin can be used when coredns is deployed on Azure or elsewhere.
+
+## Syntax
+
+~~~ txt
+azure [resource-group:dns-zone] {
+    [auth_location <path to auth_location>]
```

Comment:
we also usually don't have block quotes around these item, but say what the most minimal config looks like and then talk about extended config options (without the [ and ]) that are also possible.

Please take a look at route53/README.md and copy that.